doc/dev/development.md

```diff
@@ -143,11 +165,19 @@ git remote set-url --push upstream no_push
 
 ### Committing changes to your fork
 
-When you're happy with some changes, you can commit them and push them to your fork:
+When you're happy with some changes, you can commit them to your repo:
 
 ```shell
 git add .
 git commit
+```
+Then push the change to the fork. When prompted for authentication, using your
```

Comment:
it says fork here... that's why I was wondering - also again I think people can contribute without two factor (I think)